src/sentry/scripts/digests/digests.lua

```diff
@@ -278,10 +278,15 @@ local function digest_timeline(configuration, timeline_id)
         if redis.call('EXISTS', digest_key) == 1 then
             -- If the digest set already exists (possibly because we already tried
             -- to send it and failed for some reason), merge any new data into it.
-            -- TODO: It might make sense to trim here to avoid returning capacity *
-            -- 2 if timeline was full when it was previously digested.
```

Comment:
This was actually incorrect, it can grow to `capacity * number of digestion attempts` and the number of attempts is unbounded.